.github/workflows/test-db-ops.yaml

```
@@ -25,15 +27,144 @@
+
+      - name: Get Latest Commit ID
+        id: lastcommit
+        if: ${{ github.event_name == 'repository_dispatch' }}
+        uses: nmbgeek/github-action-get-latest-commit@main
+        with:
+          owner: ${{ github.repository_owner }}
+          repo: ballerine
+          branch: ${{ github.event_name == 'repository_dispatch' && github.event.client_payload.ref || inputs.environment }}
+
+      - name: Set Commit Id as Env 
+        run: echo "SHORT_HASH=${{ steps.lastcommit.outputs.shorthash }}" >> $GITHUB_ENV  
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:64: Double quote to prevent globbing and word splitting

(shellcheck)